georchestra-integration/georchestra-authnz/src/test/java/org/georchestra/geonetwork/security/AbstractGeorchestraIntegrationTest.java

```diff
@@ -86,7 +87,7 @@ public abstract class AbstractGeorchestraIntegrationTest {
         Testcontainers.exposeHostPorts(ldap.getMappedLdapPort(), db.getMappedDatabasePort());
 
         console = new GeorchestraConsoleContainer()//
-                .withFileSystemBind(resolveHostDataDirectory(), "/etc/georchestra")//
+                .withCopyFileToContainer(MountableFile.forClasspathResource("data_directory"), "/etc/georchestra")//
```

Comment:
why switching from a bind mount to a file copy ? (no objection, just curious)

Reply:
so to let console geotools write geojson in console datadir (to have write rights)

Reply:
:+1: